source/_components/cover.rflink.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
 title: "RFLink Cover"
-description: "Instructions on how to integrate RFLink Somfy RTS Cover into Home Assistant."
-date: 2017-08-08
+description: "Instructions on how to integrate RFLink Somfy RTS and KAKU ASUN-650 covers into Home Assistant."
+date: 2019-04-07
```

Comment:
Please do not change the `date`.

Reply:
@klaasnicolaas , My apologies, i did not know that.